core/extension/gdnative_interface.cpp

```diff
@@ -678,6 +681,59 @@ static const char32_t *gdnative_string_operator_index_const(const GDNativeString
 	return &self->ptr()[p_index];
 }
 
+static void gdnative_string_operator_plus_eq_string(GDNativeStringPtr p_self, const GDNativeStringPtr p_b) {
+	String *self = (String *)p_self;
```

Comment:
`reinterpret_cast<String*>(p_self)` instead of the C style cast ?

Reply:
Most of the casts in the `gdnative_interface.cpp` are C-casts, but I guess it might be worth changing it all for consistency.